examples/read_external_data.py

```
@@ -0,0 +1,75 @@
+    print()
+    
+    try:
+        print("4. Available data sources summary:")
+        for source in ['openaq', 'openmeteo', 'nasapower', 'firms', 'era5', 'terrain']:
+            files = manager.list_files(source)
+            if files:
+                print(f"   {source}: {len(files)} files")
+                latest = files[-1]
+                size_mb = latest.stat().st_size / (1024 * 1024)
+                print(f"      Latest: {latest.name} ({size_mb:.2f} MB)")
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)